pkg/apitype/core.go

```diff
@@ -39,7 +39,7 @@ import (
 const (
 	// DeploymentSchemaVersionCurrent is the current version of the `Deployment` schema.
 	// Any deployments newer than this version will be rejected.
-	DeploymentSchemaVersionCurrent = 1
+	DeploymentSchemaVersionCurrent = 2
```

Comment:
Does this necessitate a change to the service? (Or did it already happen?)

Reply:
This already happened earlier this week.